src/components/shared/select/Select.tsx

```diff
@@ -0,0 +1,24 @@
+import "./Select.scss";
+
+export interface SelectProps {
+  selectName: string;
+  data: string[];
+}
+
+const Select = ({ selectName, data }: SelectProps) => {
+  return (
+    <div className="select">
+      <label>{selectName}</label>
+      <div>
+        <span></span>
```

Comment:
why empty span?

Reply:
added arrow icon inside select.

Reply:
changed. inside span there is a future select default option and arrow inside button element.